lib/init.js

```diff
@@ -310,7 +313,17 @@ const lintElixirc = async (textEditor) => {
 
   const fileText = textEditor.getText();
   const execOpts = await getOpts(filePath);
+
+  const argsAsString = elixircArgs.join(' ');
+  const optsAsString = JSON.stringify(execOpts);
+  // eslint-disable-next-line no-console
+  console.log(`Executing the command: ${elixircPath} ${argsAsString} ${optsAsString}`);
```

Comment:
This debugging text should be removed or a `linter-elixirc` tag put in the text and a check for `atom.inDevMode()` wrapped around it.